### PR TITLE
Add Readme.md

### PR DESCRIPTION
**Pull request by paolojoaquinp**
## Description

#1 Solved

### CODE REVIEW

**Comment by diegoveloper** [comment]
There are some instances where you mention "animated_book_list" instead "animated_book_widget"

**Comment by paolojoaquinp** [comment]
> There are some instances where you mention "animated_book_list" instead "animated_book_widget"

Could you give me more detail about what you mean?

File: README.md [hunk 1]
```diff
@@ -1 +1,211 @@
-# animated_book_list
\ No newline at end of file
+![banner](https://i.imgur.com/ppKK1L0.jpeg)
+
+# Flutter Animated Book Widget
+
+![build](https://github.com/the-d-velopers/animated_book_list/actions/workflows/code_metrics.yml/badge.svg?branch=master)
```
Comment (diegoveloper) [inline README.md]: animated_book_list

Comment (diegoveloper) [reply README.md]: I'll change the name of the repo later

File: README.md [hunk 2]
```diff
@@ -1 +1,211 @@
...
+[![animated_book_widget](https://img.shields.io/pub/v/animated_book_list?label=animated_book_list)](https://pub.dev/packages/animated_book_list)
```
Comment (diegoveloper) [inline README.md]: same here: animated_book_list

File: README.md (hunk 1 above)
Comment (paolojoaquinp) [reply README.md]: Well, that github action info comes from the github repo. Is not from the name of the package.

File: README.md (hunk 2 above)
Comment (paolojoaquinp) [reply README.md]: I'll fix that

File: README.md [hunk 3]
```diff
@@ -1 +1,213 @@
-# animated_book_list
\ No newline at end of file
+![banner](https://i.imgur.com/ppKK1L0.jpeg)
+
+# Flutter Animated Book Widget
+
+![build](https://github.com/the-d-velopers/animated_book_list/actions/workflows/code_metrics.yml/badge.svg?branch=master)
```
Comment (monster555) [inline README.md]: The branch name should be `main`

**Review by monster555 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (paolojoaquinp) [reply README.md]: I need confirmation, which github badge URL did we use?

* with branch parameter:

Captura de pantalla 2024-01-29 a la(s) 11 01 57

* without:
Captura de pantalla 2024-01-29 a la(s) 11 05 44

Comment (diegoveloper) [reply README.md]: without the branch name

**Review by diegoveloper — APPROVED** [review]

**Pull request merged**